2001-3000/LIT2892RepCh173.xml

```diff
@@ -23,11 +23,6 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <sourceDesc>
             <p>Work of the literatures of Ethiopia and Eritrea</p>
          </sourceDesc>
-         <sourceDesc>
-            <listWit>
-               <witness/>
-            </listWit>
-         </sourceDesc>
```

Comment:
the witness has not been added back

Reply:
I was reacting to commentaries and changing the file, but I have not commited all changes yet, I am working on it still.